integration-tests/e2e/kubetest/description/1.20/working.json

```diff
@@ -165,6 +166,8 @@
   { "testcase": "[sig-storage] In-tree Volumes [Driver: ceph][Feature:Volumes] [Testpattern: Pre-provisioned PV (default fs)] subPath should support readOnly file specified in the volumeMount", "groups": ["fast"], "only": ["openstack"] },
   { "testcase": "[sig-storage] In-tree Volumes [Driver: ceph][Feature:Volumes] [Testpattern: Pre-provisioned PV (default fs)] volumes should allow exec of files on the volume", "groups": ["fast"], "only": ["openstack"] },
   { "testcase": "[sig-storage] In-tree Volumes [Driver: ceph][Feature:Volumes] [Testpattern: Pre-provisioned PV (default fs)] volumes should be mountable", "groups": ["fast"], "only": ["openstack"] },
+  { "testcase": "[sig-storage] CSI Volumes [Driver: csi-hostpath] [Testpattern: Dynamic PV (default fs)] subPath should verify container cannot write to subpath readonly volumes [Slow]","groups": ["slow"] },
+  { "testcase": "[sig-storage] PersistentVolumes NFS when invoking the Recycle reclaim policy should test that a PV becomes Available and is clean after the PVC is deleted.", "groups": ["fast"] },
```

Comment:
do you know how long these tests run?
I think we said that the `fast` group only contains tests that are faster than 2min (if I remember correctly)

Reply:
hmm, i need to confirm this by running them in single test case - so if the time is longer than 2 mins but not very long, it will be put in `slow` group?

Reply:
yes. that is because we wanted the fast group to be as fast as possible. 
Because even 2-min tests will accumulate when we run hundreds.

Reply:
hmm, actually these three test cases are less than 2 mins  (the longest one is 88 seconds)
![image](https://user-images.githubusercontent.com/42594392/117119193-49e6f480-adc4-11eb-817c-032a0421c463.png)
however i saw one of them has `slow` in title so i put it in slow - so shall i change these three all to `fast` group?

Reply:
if all are max 88sec I would put them into the fast group.
@dguendisch any objections?

Reply:
fine with me